Source/PluginEditor.cpp

```diff
@@ -45,11 +45,16 @@ DelayPluginAudioProcessorEditor::DelayPluginAudioProcessorEditor (DelayPluginAud
     delayTimeDryWetAttachment.reset(new AudioProcessorValueTreeState::SliderAttachment( valueTreeState, DelayPluginAudioProcessor::DelayTimeDryWetID,delayDryWetSlider));
 }
 
+
+
 DelayPluginAudioProcessorEditor::~DelayPluginAudioProcessorEditor()
 {
     setLookAndFeel(nullptr);
 }
 
+

```

Comment:
Minor thing but get rid of all this excess whitespace. Good practice for you to drop a commit just removing whitespace in response to this comment.